internal/cloudrunci/cloudrunci.go

```diff
@@ -177,6 +171,15 @@ func (s *Service) Request(method string, path string, opts ...func(*RetryOptions
 	return resp, lastSeen
 }
 
+// Request issues an HTTP request to the deployed service.
```

Comment:
question: I want to recap my understanding:

* Existing method: `NewRequest()` which is a request factory that allows the caller to customize the request before calling `Do()`
* New method: `Do()` which runs an HTTP request, returns a response, and manages retries.
* New method: `Request()` which wraps `NewRequest()` and `Do()` for simple HTTP requests.

We expect Request() to work for many use cases.

Reply:
`Request()` is an existing method. I updated it last week to do retries, and to make more tests use it.